pyccel/ast/builtins.py

```diff
@@ -8,13 +8,14 @@
 """
 
 from sympy import Symbol, Function, Tuple
-from sympy import Expr
+from sympy import Expr, Not
 from sympy import sympify
 from sympy.tensor import Indexed, IndexedBase
 
+import pyccel.ast.core as Core
```

Comment:
Is this how you get around your circular import problem?

Reply:
yes, I can't use this
```python
from .core import Variable
```
now it works fine. but Travis-ci failed with the Linux version

Reply:
I think that your solution to the circular import problem fools Codacy, but it does not work with Python 3.5...

Reply:
Apparently Python 3.5 supports relative circular imports: see the last bullet point [here](https://docs.python.org/3.5/whatsnew/3.5.html#other-language-changes). But you now use an absolute import to stop Codacy from complaining.